docs/writing_a_parser_plugin/parser_tests.md

```diff
@@ -0,0 +1,58 @@
+# Creating parser tests
+
+Each parser should have a series of associated (py)tests to ensure that future developments do
+not affect the intended parsing. These tests can be found under directly under  `<parserproject>/tests/`
+directory, labeled by the corresponding parser name: `test_parsername.py`.
+
+Here is an example template of such a `pytest` code:
```

Comment:
The example below is way too big. I would instead specify some good practices (and maybe keep a small example):
- Test specific cases using representative calculation. We use real-world data here. Note that this data will be public under the aforementioned license, so consult the relevant parties.
- Use `assert`ions to test proper extraction. To facilitate lookup, refer to the data in its original units.
- Make the test objective clear by choosing appropriately sized functions and parametrization.

I would also link to a tutorial and reference on `pytest`.

Reply:
agreed, I just copied your comment into a TODO for now